demonstrations/tutorial_mitigation_advantage.py

```diff
@@ -0,0 +1,276 @@
+r"""Evidence for the utility of quantum computing before fault tolerance
+========================================================================
+
+.. meta::
+    :property="og:description": Evidence for the utility of quantum computing before fault tolerance
+    :property="og:image": https://pennylane.ai/qml/_images/diffable_mitigation_thumb.png
+
+.. related::
+
+    tutorial_error_mitigation Error mitigation with Mitiq and PennyLane
+    tutorial_diffable-mitigation Differentiable quantum error mitigation
+
+*Author: Korbinian Kottmann — Posted: 16  June 2023.*
+
+Can we use contemporary quantum computers for tasks that are useful and hard to classically simulate?
+A team of scientists at IBM in [#ibm]_ claim: yes! See how they managed to faithfully estimate expectation
+values of reasonably large and reasonably deep quantum circuits using an exciting new noise aware zero noise extrapolation 
+technique for error mitigation in this demo.
+
+Introduction
+============
```

Comment:
```suggestion
```

Reply:
Or
```suggestion
Introduction
------------
```
to fix the hierarchy